perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -174,27 +177,19 @@ sub load_problem_or_display_error : Private {
     }
     elsif ( $problem->state eq 'unconfirmed' ) {
         $c->detach( '/page_error_404_not_found', [ _('Unknown problem ID') ] )
-            unless $c->cobrand->show_unconfirmed_reports ;
+            unless $c->cobrand->show_unconfirmed_reports
+            or $c->action eq 'report/confirmation';
```

Comment:
I'm afraid due to this it appears to leak the details of every unconfirmed report in the system at /report/ID/confirmation?template=confirm_problem for any unconfirmed report.

Presumably this is here for the email_sent page? I think this change needs to be revoked, and email_sent treated as a separate code-path (perhaps email sent could be a basically static page, what would be lose, only the coordinates in "Report another problem here"?)